IDEOorg/steps-bot
# #161271091 As a user, I should receive check-ins from Roo if I've completed my old work plan and my coach adds new tasks.

#### What does this PR do?
Ensures that after a new task has been added prior to an `ultimatedone` milestone, that the Roo bot responds with a check-in of the new task when `checkintopic` is texted to it.

#### Description of Task to be completed?
- update the topic when the last topic is ultimatedone and an active task exists
- update the ultimatedone.rive file to include global scripts
- write tests

#### How should this be manually tested?
- Create a client with a workplan
- Fast-forward the bot to the `checkin` topic by texting it "checkintopic". You should see a regular checkin.
- Now complete all the tasks in the coach interface by checking the checkboxes
- Fast-forward the bot to a new check-in. You should get the "ultimatedone" celebration script.
- Add a new task to the workplan
- Fast-forward the bot to a new check-in.
- Bot checks-in with the new task 

#### Any background context you want to provide?
You would need to set up the bot to work locally for manual testing. There is a ticket on that, but if necessary, consult @megame24 

#### What are the relevant pivotal tracker stories?
#161271091

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
